### PR TITLE
wr-57: Create common ErrorMessage Component

### PR DESCRIPTION
**Pull request by AnastasiaR2**
1. Component takes validation error prop and optional prop for custom styling. 
2. It has default styling that is relative to form field error styling. 
3. Applied to input and select components.

### CODE REVIEW

File: frontend/src/libs/components/select/select.tsx [hunk 1]
```diff
@@ -62,7 +63,7 @@ const Select = <
         styles={{ ...getDefaultStyles<IsMulti, Group>(), ...styles }}
         components={{ DropdownIndicator, IndicatorSeparator }}
       />
-      {hasError && <span className={cssStyles.error}>{error as string}</span>}
+      {hasError && <ErrorMessage error={error as string} />}
```
Comment (andanf-e) [inline frontend/src/libs/components/select/select.tsx]:
```suggestion
      <ErrorMessage error={error as string} />
```

File: frontend/src/libs/components/input/styles.module.scss [hunk 2]
```diff
@@ -20,7 +21,7 @@
 }
 
 .error {
-  margin-bottom: 5px;
+  margin-bottom: 0;
```
Comment (andanf-e) [inline frontend/src/libs/components/input/styles.module.scss]:
```suggestion

```

File: frontend/src/libs/components/input/styles.module.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
   width: 100%;
   height: 45px;
   margin-top: 5px;
+  margin-bottom: 22px;
```
Comment (andanf-e) [inline frontend/src/libs/components/input/styles.module.scss]: I believe you don't need this
```suggestion

```

File: frontend/src/libs/components/input/input.tsx [hunk 4]
```diff
@@ -53,11 +54,7 @@ const Input = <T extends FieldValues>({
         type={type}
         placeholder={placeholder}
       />
-      {hasError && (
-        <span className={getValidClassNames(styles.text)}>
-          {error as string}
-        </span>
-      )}
+      {hasError && <ErrorMessage error={error as string} />}
```
Comment (andanf-e) [inline frontend/src/libs/components/input/input.tsx]:
```suggestion
      <ErrorMessage error={error as string} />
```

File: frontend/src/libs/components/error-message/error-message.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  error: string;
+  className?: string;
+};
+
+const ErrorMessage: React.FC<Properties> = ({ error, className }) => {
+  return (
+    <div className={getValidClassNames(styles.errorMessage, className)}>
+      {error}
+    </div>
+  );
```
Comment (andanf-e) [inline frontend/src/libs/components/error-message/error-message.tsx]:
```suggestion
  const hasError = Boolean(error);

  return (
    <span className={getValidClassNames(styles.errorMessage, className)}>
      {hasError && error}
    </span>
  );
```

File: frontend/src/libs/components/error-message/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.errorMessage {
+  margin-top: 5px;
+  color: var(--red);
```
Comment (andanf-e) [inline frontend/src/libs/components/error-message/styles.module.scss:4]:
```suggestion
  color: var(--red);
  min-height: <desired_height>
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

**Review by andanf-e — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Review by mhrechkin — APPROVED** [review]

**Pull request merged**